index.js

```diff
@@ -113,6 +113,7 @@ const displayPost = (post) => {
     <td class="post-activity">${formatDateDiff(Date.now(), post.bumped_at)}</td>
   </tr>`;
   postsContainer.innerHTML += postRow;
+  document.getElementById("footer").style.display = "block";
```

Comment:
Nice try, but I actually have some problems with this.

Now, the function `displayPost` not only adds new post to the table, but also displays the footer.
Moreover, the `displayPost` is called on each post, so it changes the style of footer `number of posts` times.

Finally, what happens when it fails to load the posts from the server? Does it mean it's not going to display the post?

Can you find a better place for this line of code?

Reply:
Yea, that's what I imagined that it wouldn't show footer if the data doesn't load. I will give this a try again. Thanks for the feedback guys. I know I implement sheepish hacks to fix the issues

Reply:
What if you wrap the line of code inside `displayFooter` function.
Then look between lines 25-50. Can you see a suitable place for such a function?